hadoop-common-project/hadoop-common/pom.xml

```diff
@@ -393,6 +393,13 @@
       <artifactId>lz4-java</artifactId>
       <scope>provided</scope>
     </dependency>
+    <!-- will be shaded into the hadoop-common jar for RPC use only -->
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-all</artifactId>
+      <version>4.1.27.Final</version>
```

Comment:
We manage dependency versions in hadoop-project/pom.xml, so let's not specify the version here.
It's currently at 4.1.68.Final

Reply:
Removed the version.